media-video/handbrake/handbrake-1.1.1.recipe

```diff
@@ -0,0 +1,136 @@
+CHECKSUM_SHA256="e3390c5fd901fb06d72e29c62a63d373d5fb5b3467295d114d815ae7b78a9d7a"
+SOURCE_DIR="HandBrake-$portVersion"
+PATCHES="handbrake-$portVersion.patchset"
+ADDITIONAL_FILES="HandBrake.rdef"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	handbrake$secondaryArchSuffix = $portVersion
+	cmd:HandBrakeCLI
+#	app:HandBrake
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:liba52$secondaryArchSuffix
+	lib:libass$secondaryArchSuffix
+	lib:libavcodec$secondaryArchSuffix
+	lib:libbluray$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libdvdnav$secondaryArchSuffix
+	lib:libdvdread$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libfontconfig$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libfribidi$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libjansson$secondaryArchSuffix
+	lib:libmkv$secondaryArchSuffix
+	lib:libmp3lame$secondaryArchSuffix
+	lib:libmp4v2$secondaryArchSuffix
+	lib:libmpeg2$secondaryArchSuffix
+	lib:libopus$secondaryArchSuffix
+	lib:libsamplerate$secondaryArchSuffix
+	lib:libtheora$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libvpx$secondaryArchSuffix
+	lib:libx264$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:liba52$secondaryArchSuffix
+	devel:libass$secondaryArchSuffix
+	devel:libavcodec$secondaryArchSuffix
+	devel:libbluray$secondaryArchSuffix
+	devel:libbz2$secondaryArchSuffix
+	devel:libdvdnav$secondaryArchSuffix
+	devel:libdvdread$secondaryArchSuffix
+	devel:libfontconfig$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libfribidi$secondaryArchSuffix
+	devel:libharfbuzz$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libjansson$secondaryArchSuffix
+	devel:libmkv$secondaryArchSuffix
+	devel:libmp3lame$secondaryArchSuffix
+	devel:libmp4v2$secondaryArchSuffix
+	devel:libmpeg2$secondaryArchSuffix
+	devel:libogg$secondaryArchSuffix
+	devel:libopus$secondaryArchSuffix
+	devel:libsamplerate$secondaryArchSuffix
+	devel:libtheora$secondaryArchSuffix
+	devel:libvorbis$secondaryArchSuffix
+	devel:libvpx$secondaryArchSuffix
+	devel:libx264$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:cmake
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:autoconf
+	cmd:automake
+	cmd:patch
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:python2
+	cmd:m4
+	cmd:make
+	cmd:sed
+	cmd:tar
+	cmd:yasm
+	"
+
+# Please check the file state.md in this directory for a detailed description
+# of the current state of the HandBrake port
+
+PATCH()
+{
+	sed -i 's/MODULES += contrib\/.*//g' make/include/main.defs
+}
+
+BUILD()
+{
+	./configure --prefix=$prefix --force --disable-x265
+#	cd build
+#	make
+#	cd ../haiku
+#	make
```

Comment:
You have to uncomment this line (113), and delete the commented lines (110,111,112)